src/layouts/EVMLayout.vue

```diff
@@ -1,5 +1,25 @@
 <script setup lang="ts">
+import { onBeforeUnmount, onMounted } from 'vue';
+import { debounce } from 'quasar';
+
+import { usePlatformStore } from 'src/antelope';
+
 import AppNav from 'components/evm/AppNav.vue';
+
+const platformStore = usePlatformStore();
+
+const resizeListener = debounce(() => {
+    platformStore.setEvmMenuIsCollapsed(window.innerWidth < 1024);
+}, 100);
```

Comment:
when window size changes, inform the store that the nav is either collapsed or open (large desktop vs smaller screens)